packages/lean-imt/package.json

```diff
@@ -2,40 +2,14 @@
     "name": "lean-imt.sol",
     "private": true,
     "scripts": {
-        "start": "hardhat node",
-        "compile": "hardhat compile",
-        "test": "hardhat test",
+        "compile": "forge build",
+        "test": "forge test",
         "test:report-gas": "REPORT_GAS=true hardhat test",
-        "test:coverage": "hardhat coverage",
-        "typechain": "hardhat typechain",
+        "test:coverage": "forge coverage",
```

Comment:
note that the current latest version of forge does not allow excluding the `t.sol` files from coverage.
This was fixed very recently https://github.com/foundry-rs/foundry/issues/2988